rules/DeadCode/Rector/Concat/RemoveUnusedClosureVariableUseRector.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\DeadCode\Rector\Concat;
```

Comment:
this should under `Closure` namespace:

```diff
-namespace Rector\DeadCode\Rector\Concat;
+namespace Rector\DeadCode\Rector\Closure;
```

I will create new PR for it.

Reply:
I create new PR to fix namespace to `Closure`

- https://github.com/rectorphp/rector-src/pull/7432